lua/lualine/components/grapple.lua

```diff
@@ -32,9 +32,16 @@ function Component:update_status()
 
     local current = grapple.find({ buffer = 0 })
 
+    local App = require("grapple.app")
+    local app = App.get()
+    local quick_select = app.settings:quick_select()
     local output = {}
     for i, tag in ipairs(tags) do
-        local tag_str = tag.name and tag.name or i
+        -- stylus: ignore
```

Comment:
Darn, made a small typo in the suggestion

```suggestion
        -- stylua: ignore
```